frontend/src/components/User/UserList/UserList.js

```diff
@@ -149,7 +149,7 @@ const UserList = () => {
             onClick={() => handleDeregisterClick(user.id)}>
             Deregister
           </Button>
-          {user.isMaintainer ? (
+          {user.isMaintainer === 1 ? (
```

Comment:
Will it be better to assign 1 to a constant, e.g `const maintainer = 1`?

Reply:
I think this can be resolved in subsequent PR (assignment 5)